src/text-to-speech-adapter.js

```diff
@@ -34,24 +36,34 @@ function textToSpeechAdapter(conf) {
     function addTextToQueue(text) {
       if (typeof text === 'string' && text !== '') {
         ttsMessagesQueue.push(text);
-        playMessagesQueue();
+
+        var t=setInterval(function(){
+          if(finished){
+              clearInterval(t);
+              playMessagesQueue();
+          }
+        },300);
```

Comment:
Since we need to stop the incoming queues from being immediately played, we add a checking interval to let the next text be played only when previous text has completely finished playing.